Assets/Treasure/TDK/Runtime/Services/Thirdweb/TDKThirdwebService.cs

```diff
@@ -73,37 +59,74 @@ public void InitializeSDK(string chainIdentifier)
             _walletMapping = new Dictionary<string, IThirdwebWallet>();
 
             Initialized = true;
-            // v5.0.0 <<<---
+        }
+        
+        public async Task ConnectWallet(InAppWalletOptions inAppWalletOptions, int chainId) {
+            var thirdwebService = TDKServiceLocator.GetService<TDKThirdwebService>();
+            var inAppWallet = await InAppWallet.Create(
```

Comment:
@lefarchi note that we need to use `EcosystemWallet` instead.

Reply:
No need to create an instance var as Client can directly be accessed

Reply:
Good catch, i moved this from TDK.Connect.cs and missed that